compiler/rustc_ast/src/mut_visit.rs

```diff
@@ -1422,7 +1422,7 @@ pub fn noop_visit_expr<T: MutVisitor>(
             visit_opt(label, |label| vis.visit_label(label));
             vis.visit_span(span);
         }
-        ExprKind::Match(expr, arms) => {
+        ExprKind::Match(expr, arms, _) => {
```

Comment:
```suggestion
        ExprKind::Match(expr, arms, _kind) => {
```
Visitors try to be exhaustive and write all fields in the visited types explicitly.